pkg/core/engine/main.go

```diff
@@ -477,7 +477,7 @@ func (e *Engine) LoadAutoDiscovery(defaultEnabled bool) error {
 					because it may change over pipeline execution
 					such as different source version filter
 
-					Starting the id with the autodiscovery pipelinid looks enough
+					Starting the id with the autodiscovery pipelineid looks enough
```

Comment:
It's possible this should be `pipeline` or `pipelined`...

Reply:
Indeed `pipelineid` is a reference to https://github.com/updatecli/updatecli/blob/46954873f85340fafa6ea921cac61f0c5840c16a/pkg/core/config/main.go#L54